jdbc-handler/src/main/java/org/apache/hive/storage/jdbc/conf/JdbcStorageConfigManager.java

```diff
@@ -28,28 +37,38 @@
  */
 public class JdbcStorageConfigManager {
 
+  private static final Logger LOGGER = LoggerFactory.getLogger(JdbcStorageConfigManager.class);
   public static final String CONFIG_PREFIX = "hive.sql";
+  public static final String CONFIG_PWD = CONFIG_PREFIX + ".dbcp.password";
   private static final EnumSet<JdbcStorageConfig> DEFAULT_REQUIRED_PROPERTIES =
     EnumSet.of(JdbcStorageConfig.DATABASE_TYPE,
```

Comment:
this DEFAULT_REQUIRED_PROPERTIES is used for checking for params, thats what i was referring to. Now, JDBC_URL and JDBC_DRIVER_CLASS params presence is not checked with this change.